api/HouseOutput.cs

```diff
@@ -11,15 +19,101 @@ namespace api
  */
 public class HouseOutput : IDeviceOutput
 {
-	public HouseOutput()
+	public HouseOutput(string house_info, string device_info)
 	{
+		_houseInfo = house_info;
+		_deviceInfo = device_info;
 	}
 
 	public bool write(Device dev)
 	{
-		//TODO: Implement writing to the house
+		string json = JsonConvert.SerializeObject(dev);
+		string houseID = dev.ID.HouseID.ToString();
+		string roomID = dev.ID.RoomID.ToString();
+		string deviceID = dev.ID.DeviceID.ToString();
+
+		var _ = writeHelper(json, houseID, roomID, deviceID);
```

Comment:
I'm pretty sure setting a variable to what the writeHelper function returns works the same way as an await. And then I don't have to make the write function async and make it return a task.